cmd/cleantool/cleantool.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v52/github"
+	"golang.org/x/oauth2"
+)
+
+func isPackageToBeClean(p *github.PackageVersion) bool {
+	daysBack := 90
+	toBeClean := false
+
+	if time.Now().After(p.UpdatedAt.Add(time.Duration(daysBack) * 24 * time.Hour)) {
+		log.Printf("Stale version: %v (%v, %s)", p.GetID(), p.GetVersion(), p.UpdatedAt)
```

Comment:
`p.GetVersion()` is always empty from what I see, do you need this?

I'm seeing
```
2023/05/02 14:40:14 Stale version: 20867700 (, 2022-04-28 13:48:57 +0000 UTC)
2023/05/02 14:40:14 Stale version: 20867694 (, 2022-04-28 13:48:56 +0000 UTC)
2023/05/02 14:40:14 Stale version: 20854717 (, 2022-04-28 11:46:28 +0000 UTC)
2023/05/02 14:40:14 Stale version: 20854715 (, 2022-04-28 11:46:27 +0000 UTC)
```